test/integration/mockiavelli.int.test.ts

```diff
@@ -706,4 +706,20 @@ describe.each(CONTROLLERS)(`Mockiavelli integration (%s)`, (controller) => {
         });
         await mock.waitForRequestsCount(2);
     });
+
+    test('disable() method disables mocking of requests', async () => {
+        const mockedFun = jest.fn().mockReturnValue({ status: 200 });
+        await ctx.mockiavelli.mockGET('/example', mockedFun);
+
+        await ctx.makeRequest('GET', '/example');
+        expect(mockedFun).toHaveBeenCalledTimes(1);
```

Comment:
instead of relying on mock, use public API for assertions like in other tests. For example, by asserting on `response.status`,

Reply:
Ok, changed :)